finagle-http/src/test/scala/com/twitter/finagle/http/filter/StatsFilterTest.scala

```diff
@@ -25,13 +26,20 @@ class StatsFilterTest extends FunSuite {
 
     Time.withCurrentTimeFrozen { _ =>
       Await.result(filter(Request()))
+      Await.result(filter(Request()))
     }
 
-    assert(receiver.counters(Seq("status", "404")) == 1)
-    assert(receiver.counters(Seq("status", "4XX")) == 1)
-    // TODO: until we can mock stopwatches
-    //      receiver.stats(Seq("time", "404"))      must_== Seq(0.0)
-    //      receiver.stats(Seq("time", "4XX"))      must_== Seq(0.0)
-    assert(receiver.stats(Seq("response_size")) == Seq(5.0))
+    // Verify that the counters and stats were only created once
+    verify(receiver, times(1)).counter("status", "404")
+    verify(receiver, times(1)).counter("status", "4XX")
+    verify(receiver, times(1)).stat("time", "404")
+    verify(receiver, times(1)).stat("time", "4XX")
+    verify(receiver, times(1)).stat("response_size")
+
+    assert(receiver.counters(Seq("status", "404")) === 2)
```

Comment:
please use ==, instead of ===.  scalatest has an assert macro which does the right thing here.